src/modules/emote_menu/twitch/EmoteMenu.jsx

```diff
@@ -85,19 +85,16 @@ export default class EmoteMenuModule {
       );
       mountedNode = buttonContainer;
     }
-
-    this.show(true);
   }
 
   show(visible) {
     const legacyContainer = document.querySelector(LEGACY_BTTV_EMOTE_PICKER_BUTTON_CONTAINER_SELECTOR);
-    const emoteMenuEnabled = settings.get(SettingIds.EMOTE_MENU);
```

Comment:
why did you remove this (and the above)? there is no way to disable the emote menu as-is. we can probably update the above to be:

```
if (!emoteMenuEnabled) {
  this.show(false);
}

```